testingutils/src/test/java/com/github/awsjavakit/testingutils/networking/WiremockHttpClientTest.java

```diff
@@ -0,0 +1,59 @@
+package com.github.awsjavakit.testingutils.networking;
```

Comment:
Since I wrote the test for ApiRequestBuilder, Jacoco complains about test coverage, so I needed to fix this as well.